share/doc/xml/aplimits.xml

```diff
@@ -0,0 +1,347 @@
+        counts and areas are determined from those pixels whose centers fall
+        within the aperture. We recommend the use of dmextract if an event list
+        is available.
+      </PARA>
+      <PARA>
+        Unfortunately, determination of effective exposure and psf fractions in
+        apertures will almost always require the use of exposure map and psf
+        images, leading to possible inaccuracies in the determination of these
+        quantities. For apertures containing many image pixels, this is a
+        negligible effect, because exposure maps typically vary smoothly, and
+        thus inclusion or exclusion of particular exposure map pixels has little
+        effect on the average in the aperture. Similarly, the psf fraction
+        change due to inclusion or exclusion of a few pixels at the aperture
+        edge is likely to be small.
+      </PARA>
+      <PARA>
+        This will not be the case, however, if aperture sizes are small compared
+        to image pixel sizes. In those cases, it is recommended that the user
+        repixelate the psf and exposure map images to a finer scale.
+      </PARA>
+      <PARA>
+        This tool itself does not account for the fraction of the PSF that falls
+        outside the source aperture, but one can divide the resulting upper flux
+        limit by the PSF fraction in the aperture to estimate the upper limit on
+        the count rate.
+      </PARA>
+      <PARA title="2.) Upper limit on a detection vs. credible interval for the flux">
+        This tool calculates the upper limit on the source flux, given
+        properties of the detection method (type I and II errors and source
+        region size and exposure time) and background level. This calculation
+        true for all source region in an observations (assuming a smooth
+        background and constant exposure time). This is conceptually different
+        from aprates, which instead looks at the source count rate in one
+        specific region and then derives the credible interval for the flux of a
+        source at one specific region. For undetected or weak sources, the
+        credible interval includes 0 and sometimes the upper end of the interval
+        is taken as upper limit on the source flux. That is not mathematically
+        correct, since "upper limit" and "upper end of credible interval"
+        describe different statistical concepts, even though in the derived
+        numbers are can be similar; see
+        <HREF link="https://ui.adsabs.harvard.edu/abs/2010ApJ...719..900K">Kashyap et al. (2010)</HREF>
+        for a more detailed discussion.
+      </PARA>
+    </DESC>
+
+    <QEXAMPLELIST>
+      <QEXAMPLE>
+        <SYNTAX>
+          <LINE>unix% aplimits typeIerror=.1 typeIIerror=.5 bkg_rate=1.23</LINE>
+          <LINE>unix% pget aplimits_out.par upper_limit</LINE>
+          <LINE>2.44206</LINE>
+          <LINE>3</LINE>
```

Comment:
Just checking -- Is this example right?  It's only `pget`'ing the `upper_limit` value, but you show two values are returned.